api/catalog/api/models/media.py

```diff
@@ -142,7 +142,7 @@ class AbstractMediaReport(models.Model):
     deleted_class: type[models.Model] = None
     """the class storing deleted media e.g. ``DeletedImage`` or ``DeletedAudio``"""
 
-    BASE_URL = "https://search.creativecommons.org/"
+    BASE_URL = "https://wordpress.org/openverse/"
```

Comment:
Here you can use the new variable from the settings like this:
```suggestion
    BASE_URL = settings.BASE_URL
```